app/controllers/ontology_search_controller.rb

```diff
@@ -2,12 +2,17 @@ class OntologySearchController < ApplicationController
 
   def search
     @search_response = paginate_for(search_response)
-    render 'ontologies/_ontology_search'
+    render 'shared/_ontology_search'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.